app/views/articles/_sticky_nav.html.erb

```diff
@@ -53,4 +53,17 @@
       </div>
     <% end %>
   <% end %>
+
+  <% cache(release_adjusted_cache_key("article-sidebar-display-ad-#{params[:timeframe]}-#{user_signed_in?}"), expires_in: (params[:timeframe].blank? ? 120 : 360).seconds) do %>
```

Comment:
Out of everything here, I'm probably least confident in & would most greatly appreciate :eyes: on this `cache` instruction, which I've mostly cargo-culted from other areas with display ads.

Reply:
I'm sort of trying to figure this out as well [here](https://github.com/forem/forem/pull/18695) and maybe we want the "post_comments" and "post_sidebar" to have the same cache keys?  

What are the `params[:timeframe]` within the context of a single article on a show page? In an articles index context I think they refer to the timeframes that we filter by (i.e. `week`, `month`, `year`, `infinity` etc.)

Reply:
I updated this in 7a4f03e, to remove `params[:timeframe]` and make the cache key correspond to https://github.com/forem/forem/pull/18695 -- with a slightly distinct prefix (`article-sidebar-content-#{rand(5)}` vs `article-bottom-content-#{rand(5)}`)